server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/routes/UserQuotaRoutes.java

```diff
@@ -90,6 +98,19 @@ private static Optional<Integer> intQueryParameter(Request request) {
                     "strictly positive optional integer", USERS_PER_SECOND), e);
             }
         }
+
+        private static Optional<QuotaComponent> getQuotaComponent(Request request) {
+            String quotaComponentString = request.queryParams(USERS_PER_SECOND);
+            if (Objects.isNull(quotaComponentString)) {
+                return Optional.empty();
+            }
+            QuotaComponent quotaComponent = QUOTA_COMPONENT_MAP.get(request.queryParams(USERS_PER_SECOND));
```

Comment:
Idem... and actually it's supposed to have been extracted already in `quotaComponentString`